### PR TITLE
Cache scripts locally

### PR DESCRIPTION
**Pull request by tengqm**
The current layout partial is using scripts from a site not accessible
from behind the dam great firewall. This is a fix to cache the scripts
as we do before so that tabs, top menu works for everyone.

closes: #26286

### CODE REVIEW

**Comment by kbhawkey** [comment]
👀

**Comment by kbhawkey** [comment]
Hi @tengqm .
I'd like to better understand the issue.
For any web client in China, the site is not displaying the tooltip or tab components?

**Comment by tengqm** [comment]
@kbhawkey Any web behavior that depends on the bootstrap JS are not working: the main dropdown menu, the tab list and things like that.

**Comment by tengqm** [comment]
The `layouts/partials/scripts.html` is copied from `./themes/docsy/layouts/partials/scripts.html`. In other words, by overriding the builtin partial, things are working now.

**Comment by kbhawkey** [comment]
> 
> 
> @kbhawkey Any web behavior that depends on the bootstrap JS are not working: the main dropdown menu, the tab list and things like that.

Okay. Docs will need to maintain the versions of mermaid.js, bootstrap.js, and popper.js.
/lgtm

**Comment by celestehorgan** [comment]
I'm going to have to take your word for it since I can't test this out (for obvious reasons), @tengqm 

/lgtm 

Asking @sftim or @onlydole for an approve if possible!

**Comment by tengqm** [comment]
For reviewers who cannot reproduce the problem, here is a screenshot for you to understand what is happening from behind the damn wall:

image

File: layouts/partials/scripts.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
+
+<script src="/js/popper-1.14.3.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous"></script>
+<script src="/js/bootstrap-4.3.1.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+
+{{ if .Site.Params.mermaid.enable }}
+<script src="/js//mermaid.min.js" crossorigin="anonymous"></script>
```
Comment (sftim) [inline layouts/partials/scripts.html:6]: nit:
```suggestion
<script src="/js/mermaid.min.js" crossorigin="anonymous"></script>
```

(but it's just a nit - the code as committed works OK)

**Review by sftim — COMMENTED** [review]
I'd call this “vendoring”, not cacheing. But yes, looks fine.
/approve

**Comment by tengqm** [comment]
Thanks for approving this. The problem is resolved.

**Pull request merged**